packages/flutter_tools/lib/src/devfs.dart

```diff
@@ -566,6 +566,7 @@ class DevFS {
     AssetBundle? bundle,
     bool bundleFirstUpload = false,
     bool fullRestart = false,
+    bool resetCompiler = false,
```

Comment:
I see that this API has no docs for all these arguments but since you are adding one now, does `resetCompiler` mean send the reset signal to the frontend server?

Also what does `fullRestart` mean now?

Reply:
Indeed, `fullRestart` means the same as before - we were just using it to also signal whether we do the reset. I know the parameters aren't documented so it looks weird, but I think adding docs for the parameters I'm adding/modifying makes sense.